solutions/src/main/java/ru/sberbank/school/task09/RouteServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package ru.sberbank.school.task09;
+
+import lombok.NonNull;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by Mart
+ * 18.06.2019
+ **/
+public class RouteServiceImpl extends RouteService<City, Route<City>> {
+    private File caheFile;
+    private RouteSerializableManager serializableManager;
+
+    public RouteServiceImpl (@NonNull String filepath, RouteSerializableManager serializableManager) {
```

Comment:
&#39;(&#39; is preceded with whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1686201202>*